civihr_default_theme/template.php

```diff
@@ -481,10 +485,14 @@ function civihr_default_theme_js_alter(&$javascript) {
       'sites/all/modules/civihr-contrib-required/fancy_login/js/fancy_login.js',
       'sites/all/themes/civihr_employee_portal_theme/civihr_default_theme/assets/js/radix.modal.js',
       'sites/all/modules/civicrm/packages/jquery/plugins/jquery.blockUI.min.js',
-      'https://sdk.yoti.com/clients/browser.js',
       'sites/all/modules/civihr-signup/js/civihr-signup.js',
       'https://use.typekit.net/mhr5yod.js',
     ];
+
+    if (module_exists('yoti')) {
+      array_push($requiredJS, YotiHelper::YOTI_SDK_JAVASCRIPT_LIBRARY);
+    }
```

Comment:
There's no need for calling `module_exists` twice. Please move the require_once from line 17 to this if. Also, the usual way to add an item to the end of an array in PHP is to use the `[]` operator, not `array_push`:

```php
$requiredJS[] = YotiHelper::YOTI_SDK_JAVASCRIPT_LIBRARY;
```